txpool/pool_test.go

```diff
@@ -152,24 +153,24 @@ func TestPrepareBlockTransactions(t *testing.T) {
 	val3.AddToStake(10000000000)
 	td.sandbox.UpdateValidator(val3)
 
-	transferTx := tx.NewTransferTx(block1000000.Stamp(), acc1.Sequence()+1, acc1Signer.Address(),
```

Comment:
To make the test better, let's define `randHeight` and `randBlock`, then `lockTime` can be `randHeight+1` etc.